src/components/HomePage/index.js

```diff
@@ -11,16 +11,38 @@ import ListItemText from "@material-ui/core/ListItemText";
 import List from "@material-ui/core/List";
 import { userList } from "./userList";
 import useStyles from "./styles";
+import SideBar from "../SideBar/index";
 
 function HomePage({ background = "white", textColor = "black" }) {
   const classes = useStyles();
-  const [value, setValue] = useState(2);
+    const [value, setValue] = useState(2);
+    const [footerContent, setFooterContent] = useState([
```

Comment:
is this component gets used in many places?

Reply:
Not as of now. But considering the possibility of using floating links, I have tried to make it dynamic.

Reply:
so shall we export the entire useEffect to another standalone component?

https://reactjs.org/docs/hooks-custom.html